app/views/search/results.html.erb

```diff
@@ -34,11 +34,7 @@
       </ul>
   </p>
 
-  <% if params[:advanced].present? %>
-    <%= render partial: "advanced_search/form" %>
-  <% else %>
-    <%= render partial: "basic_search/form" %>
-  <% end %>
+  <%= render partial: "basic_search/form" %>
```

Comment:
The partial being rendered here should likely move to `search/_form` now that it handles both